### PR TITLE
Fix #3365: [RTL] High-fi All general buttons

### LINKED ISSUE #3365: [RTL] High-fi All general buttons
In Exploration Player all the general button `Return to topic`, `submit`, `previous`, `next` `replay` and  `continue` button need to support RTL view.

screenshot

### PR DESCRIPTION
**Pull request by veena14cs**
## Explanation
This PR fixed #3365 . This PR fixes the alignment of all the general button `Return to topic`, `submit`, `previous`, `next`, `replay` and `continue` button  button in RTL.

## Screenshot LTR and RTL

**Mobile** 
screenshot ........ screenshot
screenshot ........ screenshot

screenshot.....screenshot

screenshot......screenshot

screenshot.....screenshot

screenshot.......screenshot

screenshot......screenshot

screenshot.....screenshot

**Tablet**

screenshot  ........ screenshot

screenshot.....screenshot

screenshot.......screenshot

screenshot.......screenshot

screenshot...........screenshot

screenshot......screenshot

screenshot......screenshot

screenshot.....screenshot

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/res/layout/return_to_topic_button_item.xml [hunk 1]
```diff
@@ -14,9 +14,8 @@
   <androidx.constraintlayout.widget.ConstraintLayout
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:gravity="center_vertical"
-    app:layoutMarginEnd="@{buttonViewModel.hasConversationView ? @dimen/general_button_item_exploration_view_margin_end : @dimen/general_button_item_question_view_margin_end}"
-    app:layoutMarginStart="@{buttonViewModel.hasConversationView ? @dimen/general_button_item_exploration_view_margin_start : @dimen/general_button_item_question_view_margin_start}"
+    android:paddingEnd="@{buttonViewModel.hasConversationView ? @dimen/general_button_item_exploration_view_margin_end : @dimen/general_button_item_question_view_margin_end}"
+    android:paddingStart="@{buttonViewModel.hasConversationView ? @dimen/general_button_item_exploration_view_margin_start : @dimen/general_button_item_question_view_margin_start}"
```
Comment (rt4914) [inline app/src/main/res/layout/return_to_topic_button_item.xml]: I think you will need to apply this to other buttons too like `continue_button`, `previous_button`, `next_button` etc. 
I suggest fixing all those buttons in single PR as you will need to rename the dimens too considering these are not margins anymore, instead they are padding.

`general_button_item_exploration_view_margin_end ` -> `general_button_item_exploration_view_padding_end `

**Review by rt4914 — CHANGES_REQUESTED** [review]
@veena14cs PTAL

File: app/src/main/res/layout/return_to_topic_button_item.xml (hunk 1 above)
Comment (veena14cs) [reply app/src/main/res/layout/return_to_topic_button_item.xml]: Added padding dimensions in dimens file. `continue_button`, `previous_button`, `next_button`, and `submit_button` are correct in RTL. I have added screenshots in description for reference.

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**